src/vs/workbench/contrib/chat/browser/chatWidget.ts

```diff
@@ -794,7 +794,14 @@ export class ChatWidget extends Disposable implements IChatWidget {
 			);
 
 			let welcomeContent: IChatViewWelcomeContent;
+			// --- Start Positron ---
+			/*
 			if ((startupExpValue === StartupExperimentGroup.MaximizedChat
+			*/
+			// Don't use the experimental view for Positron Assistant
+			const useExpView = false;
+			if (useExpView && (startupExpValue === StartupExperimentGroup.MaximizedChat
+				// --- End Positron ---
```

Comment:
Not sure we can actually get into this experimental view codepath, but just to be sure, I've made it so we always bail out of this

Reply:
I'm surprised that the linter doesn't complain about `useExpView` always being `false` and creating an inaccessible code block.